modules/infra/main.tf

```diff
@@ -100,7 +100,7 @@ locals {
 }
 
 provider "aws" {
-  region = "us-east-1"
+  region = strcontains(var.region, "us-gov") ? "us-gov-east-1" : "us-east-1"
```

Comment:
this just seems strange and wrong given its for a specific purpose but the alias doesn't provide any inkling.
Does the service actually exist in gov cloud?

Reply:
This service only exist in us-east-1 for commercial accounts, given the alias refers to that zone and we can not condition the provider.

Reply:
us-east-1 and us-gov-east-1 are not the same.

Reply:
what confuses me more is something line line 14. And that won't work for gov-cloud.

Reply:
it wont work given it only exists in us-east-1, given that we need to give the provider something  I guess i could give it the `var.region` and short circuit the module so that its off in a gov region.
```hcl
module "cost_usage_report" {
  count        = var.domino_cur.provision_cost_usage_report && !strcontains(var.region, "us-gov") ? 1 : 0
```

Reply:
I am not sure if its possible at least not straight forward, will bring it up with the CUR folks. for now I am just circuiting the module